kafka_quota_enforcer/src/main/java/com/tesla/data/quota/enforcer/QuotaEnforceCommand.java

```diff
@@ -36,13 +36,15 @@ protected Enforcer<ConfiguredQuota> initEnforcer() {
     Map<String, Object> zkConfig = zookeeperConfig();
     this.zkClient = KafkaZkClient.apply(
         zkConfig.get("connect").toString(),
-        JaasUtils.isZkSecurityEnabled(),
+        // NOTE: The function below only checks if SASL is enabled, it does not check if TLS is enabled.
+        JaasUtils.isZkSaslEnabled(),
```

Comment:
The older function, `JaasUtils.isZkSecurityEnabled()` was also checking for SASL only, not TLS. So this is not a regression in this PR.